src/server/src/checkstyle/java.header

```diff
@@ -1,16 +1,17 @@
-/*
-( * Copyright 2014-2017 Spotify AB)?( * Copyright \d\d\d\d-\d\d\d\d Stefan Podkowinski)?
-( * Copyright \d\d\d\d-\d\d\d\d The Last Pickle Ltd)?
- *
- * Licensed under the Apache License, Version 2.0 \(the "License"\);
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+\/\*
+( \* Copyright 2014-2017 Spotify AB)?( \* Copyright \d\d\d\d-\d\d\d\d Stefan Podkowinski)?
+( \* Copyright \d\d\d\d-\d\d\d\d The Last Pickle Ltd)?
+( \* Copyright \d\d\d\d-\d\d\d\d DataStax, Inc.)?
+( \*)?(\* Licensed under the Apache License, Version 2.0 \(the "License"\);)?
+( \* Licensed under the Apache License, Version 2.0 \(the "License"\);)?( \* you may not use this file except in compliance with the License\.)?
+( \* you may not use this file except in compliance with the License\.)?( \* You may obtain a copy of the License at)?
+( \* You may obtain a copy of the License at)?( \*)?
+( \*)?( \*     http:\/\/www\.apache\.org\/licenses\/LICENSE-2\.0)?
+( \*     http:\/\/www\.apache\.org\/licenses\/LICENSE-2\.0)?( \*)?
+( \*)?( \* Unless required by applicable law or agreed to in writing, software)?
+( \* Unless required by applicable law or agreed to in writing, software)?( \* distributed under the License is distributed on an "AS IS" BASIS,)?
+( \* distributed under the License is distributed on an "AS IS" BASIS,)?( \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.)?
+( \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.)?( \* See the License for the specific language governing permissions and)?
+( \* See the License for the specific language governing permissions and)?( \* limitations under the License\.)?
+( \* limitations under the License\.)?( \*)?
+( \*)?
```

Comment:
this makes the license header check meaningless.

Reply:
> Can the `src/` doc additions be in the original commit, and the generated `doc/` stuff be in a separate "generate doc" commit?

Good point

Reply:
Does it? I'm trying to match the new case where you can have 3 lines of copyright. Checkstyle header checks kinda sucks with this as we can't have optional lines easily.
Here I'm trying to make it so that one line could end up having one pattern or the other, like we did for the Spotify/Podkowinsky copyright line.
Why would it make it meaningless then?

Reply:
I think I found a proper set of regex to match all the cases we have in the code now.
Man we're starting to have too many copyrights in there 😂

Reply:
ok, so my initial fix was wrong.
I used a single pattern for the copyright line which allows the known names we're expecting to find there, and made that line repeatable in `checkstyle.xml`.

It looks much better now and we can extend the list of copyrights again in the future if needed :D